index.html

```diff
@@ -279,7 +279,7 @@ <h6 class="description">Hello I’m a software developer! I can help you build a
     </section>
   </div>
   <footer class="form-cont" id="form-cont">
-  <form action="https://formspree.io/f/mrgdkyla" method="post"  class="form">
+  <form class="form">
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you follow the Figma design template by ensuring that the text on the left side of your form is aligned like it is in the Figma Design.



|  |  |
| ---      | ---       |
| Your Design | <img width="930" alt="image" src="https://user-images.githubusercontent.com/52934128/197276551-c5e852db-1c39-4348-8a5c-647df964ab0a.png">         |
| Figma Design    | <img width="550" alt="image" src="https://user-images.githubusercontent.com/52934128/197276614-84541561-e55e-4c4b-8372-fe55a2384432.png">      |